lumibot/backtesting/backtesting_broker.py

```diff
@@ -468,55 +468,48 @@ def process_pending_orders(self, strategy):
                 low = ohlc.df["low"][-1]
                 close = ohlc.df["close"][-1]
                 volume = ohlc.df["volume"][-1]
-                
+
             #############################
             # Determine transaction price.
             #############################
-            
+
             if order.type == "market":
-                price = close
-                
+                price = open
+
             elif order.type == "limit":
                 price = self.limit_order(
-                    order.limit_price, order.side, close, high, low
+                    order.limit_price, order.side, open, high, low
                 )
-                
+
             elif order.type == "stop":
-                price = self.stop_order(order.stop_price, order.side, close, high, low)
-                
+                price = self.stop_order(order.stop_price, order.side, open, high, low)
+
             elif order.type == "stop_limit":
                 if not order.price_triggered:
                     price = self.stop_order(
-                        order.stop_price, order.side, close, high, low
+                        order.stop_price, order.side, open, high, low
                     )
-                    if price != None:
+                    if price:
```

Comment:
wouldn't this also filter out $0 fills?

Reply:
Technically yes ... but NYSE/NASDAQ/etc don't allow any $0 prices, right?

Reply:
Reverted back to using None